features/support/selenium_fallback.rb

```diff
@@ -0,0 +1,9 @@
+Around('@selenium') do |scenario, block|
```

Comment:
Unused block argument - `scenario`. If it's necessary, use `_` or `_scenario` as an argument name to indicate that it won't be used.